### ISSUE TITLE
got time successfully but no change

### ISSUE THREAD

**Issue by ndemirci** [uxDaniel/visa_rescheduler#69]
it said "got the time successfully" but no change in the appointment. Do you know why this could be happen?
I am using Soroosh-N version btw. 

<img width="491" alt="image" src="https://github.com/uxDaniel/visa_rescheduler/assets/134543350/8554ce65-7c4e-4c30-bcfe-a89e62b52257">

**Comment by ndemirci** [comment]
it happend twice. weird part is the date and time were same with the first one. I could not understand the issue. Is this fake time?

<img width="554" alt="image" src="https://github.com/uxDaniel/visa_rescheduler/assets/134543350/271451fe-ac77-4b26-9574-3be5ea4192bf">

**Comment by ndemirci** [comment]
<img width="565" alt="image" src="https://github.com/uxDaniel/visa_rescheduler/assets/134543350/43550506-ee2a-4286-9ab0-f309a3365df3">

did it again so there has to be a problem I need to solve. Otherwise I can not reschedule...

**Comment by KaranmaanCanada** [comment]
> <img alt="image" width="565" src="https://user-images.githubusercontent.com/134543350/279582634-43550506-ee2a-4286-9ab0-f309a3365df3.png">
> did it again so there has to be a problem I need to solve. Otherwise I can not reschedule...

Did you see these slots yesterday? And do you use the script on cloud?

**Comment by ndemirci** [comment]
> > <img alt="image" width="565" src="https://user-images.githubusercontent.com/134543350/279582634-43550506-ee2a-4286-9ab0-f309a3365df3.png">
> > did it again so there has to be a problem I need to solve. Otherwise I can not reschedule...
> 
> Did you see these slots yesterday? And do you use the script on cloud?

Yes I do except the first one which I needed for. However in my country these kind of slots not being so long. It is taken by someone in 30 sec-1 min. So havent seen the first and early one is totally normal.

I use my own computer, not on cloud

**Comment by KaranmaanCanada** [comment]
> > > <img alt="image" width="565" src="https://user-images.githubusercontent.com/134543350/279582634-43550506-ee2a-4286-9ab0-f309a3365df3.png">
> > > did it again so there has to be a problem I need to solve. Otherwise I can not reschedule...
> > 
> > 
> > Did you see these slots yesterday? And do you use the script on cloud?
> 
> Yes I do except the first one which I needed for. However in my country these kind of slots not being so long. It is taken by someone in 30 sec-1 min. So havent seen the first and early one is totally normal.
> 
> I use my own computer, not on cloud

Which country are you in ?

**Comment by ndemirci** [comment]
> > > > <img alt="image" width="565" src="https://user-images.githubusercontent.com/134543350/279582634-43550506-ee2a-4286-9ab0-f309a3365df3.png">
> > > > did it again so there has to be a problem I need to solve. Otherwise I can not reschedule...
> > > 
> > > 
> > > Did you see these slots yesterday? And do you use the script on cloud?
> > 
> > 
> > Yes I do except the first one which I needed for. However in my country these kind of slots not being so long. It is taken by someone in 30 sec-1 min. So havent seen the first and early one is totally normal.
> > I use my own computer, not on cloud
> 
> Which country are you in ?

turkey

**Comment by KaranmaanCanada** [comment]
> > > > > <img alt="image" width="565" src="https://user-images.githubusercontent.com/134543350/279582634-43550506-ee2a-4286-9ab0-f309a3365df3.png">
> > > > > did it again so there has to be a problem I need to solve. Otherwise I can not reschedule...
> > > > 
> > > > 
> > > > Did you see these slots yesterday? And do you use the script on cloud?
> > > 
> > > 
> > > Yes I do except the first one which I needed for. However in my country these kind of slots not being so long. It is taken by someone in 30 sec-1 min. So havent seen the first and early one is totally normal.
> > > I use my own computer, not on cloud
> > 
> > 
> > Which country are you in ?
> 
> turkey

Did you find anything solution?

**Comment by Arsh20101** [comment]
> > > > > > <img alt="image" width="565" src="https://user-images.githubusercontent.com/134543350/279582634-43550506-ee2a-4286-9ab0-f309a3365df3.png">
> > > > > > did it again so there has to be a problem I need to solve. Otherwise I can not reschedule...
> > > > > 
> > > > > 
> > > > > Did you see these slots yesterday? And do you use the script on cloud?
> > > > 
> > > > 
> > > > Yes I do except the first one which I needed for. However in my country these kind of slots not being so long. It is taken by someone in 30 sec-1 min. So havent seen the first and early one is totally normal.
> > > > I use my own computer, not on cloud
> > > 
> > > 
> > > Which country are you in ?
> > 
> > 
> > turkey
> 
> Did you find anything solution?

Remove the UTF-8 headers, I have made some improvements will commit and publish as a fork, I am also adding countdown timer and also the ability to keep retrying

**Comment by Ali-Flt** [comment]
@Arsh20101, can you be more specific on what is the fix for this? Also, did you make the fork?
